resources/views/includes/featuredProducts.blade.php

```diff
@@ -0,0 +1,45 @@
+<section class="featured-product section-padding">
+    <div class="container">
+        <div class="row">
+            
+            <div class="col-12 text-center">
+                <h2 class="mb-5">Featured Products</h2>
+            </div>
+            @foreach ($products as $product)
+                
+            
+            <div class="col-lg-4 col-12 mb-3">
+                <div class="product-thumb">
+                    <a href="product-detail.html">
+                        <img src="{{ asset('assets/images/' .$product->image) }}" class="img-fluid product-image" alt="">
```

Comment:
look at the path `assets/images/` and you make in FashionFactory `asset\images\products\`